configs/default/IFRC-IFLIGHT_BLITZ_F405.config

```diff
@@ -0,0 +1,92 @@
+# Betaflight / STM32F405 (S405) 4.2.9 Apr 27 2021 / 19:33:01 (e097f4ab7) MSP API: 1.43
```

Comment:
As I said before, this is not the correct commit. Please export config from a betaflight official unified target not some other third party one.

Reply:
My "employment relationship" is non of your business, neither should this be of your concern approving pull requests lol.
Since you and iFlight both are Chinese nationals, I'll let you two grown ups do the talking and I'll pull back at this point!
This commit was sent to me by iFlight to correct the last "wrong" commit. Seems it was wrong again...